config/secrets.yml

```diff
@@ -19,4 +19,4 @@ test:
 # Do not keep production secrets in the repository,
 # instead read values from the environment.
 production:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_key_base: cbac2a461d1dd504876c34a879120f485c526762e341ab09840289fa5372c2ae70d87adde98eb263bd78909bf6457a4f457cbf77b7df71741228bcca7ab485d9
```

Comment:
any particular reason for moving this out of environment?

Reply:
Just copied the one from dev to prod. Will change back to using prod's key in Env! Thanks!